test/spec/app_spec.rb

```diff
@@ -783,10 +797,51 @@
     #   skip("Needs to be implemented")
     # end
 
-    # it "should edit team logo" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should edit team and logo", bin1: true do
+      team = "testteam#{Time.now.to_i}"
+      api_create_team(team:team)
+      p = Page.new(config: @config, driver: @driver)
+      p.go(@config['self_url'] + '/' + team)
+      wait_for_selector("team-members__edit-button", :class)
+      str= @driver.page_source
+      str = str[str.index('team__edit-button')..str.length]
+      str = str[0..str.index('tabindex') - 3]
+      str = "." +str.gsub(" ", ".")
```

Comment:
Same question as before: what's the idea of this block that goes from line 806 until here?

Reply:
Same answer as before.